ci/docker/os-image-stemcell-builder/Dockerfile

```
@@ -1,126 +1,124 @@
+    && echo "${OVF_TOOL_INSTALLER_SHA1} /tmp/ovftool_installer.bundle" | sha1sum -c - \
+    && chmod a+x ./ovftool_installer.bundle \
+    && ln -s /bin/cat /usr/local/bin/more \
+    && bash  ./ovftool_installer.bundle --eulas-agreed \
+    && rm -rf ./ovftool_installer.bundle /tmp/vmware-root/ /usr/local/bin/more
+
+RUN meta4_cli_path="/usr/local/bin/meta4" \
+    && curl --show-error -sL "${META4_CLI_URL}" \
+      > "${meta4_cli_path}" \
+    && chmod +x "${meta4_cli_path}"
+
+RUN cd /tmp  \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)